crypto/sm2/sm2_crypt.c

```diff
@@ -112,11 +114,17 @@ int SM2_encrypt(const EC_KEY *key,
     kG = EC_POINT_new(group);
     kP = EC_POINT_new(group);
     if (kG == NULL || kP == NULL)
+       {
```

Comment:
you are using the old style ...
this brace should be aligned just below the 'i' of if

Reply:
If I use `util/openssl-format-source` script it formats this as
```
    if (kG == NULL || kP == NULL) {
        SM2err(SM2_F_SM2_ENCRYPT, ERR_R_MALLOC_FAILURE);
        goto done;
    }
```
Is this script out of date?

Reply:
No , it isn't so out ... ;)
It just has some issues like with #define like STACK_OF(TYPE) * ptr ...
The style guide was documented, take a look at existing code.
This kind of accolade placement was used a long time ago...

Reply:
Fixed the brace placement, thanks

Reply:
Unfortunately the brace placement is still wrong! I think @FdaSilvaYY got confused above. The correct brace placement is on then end of a line as you show in your example @randombit.